sklearn/metrics/_pairwise_distances_reduction/_radius_neighborhood.pxd.tp

```diff
@@ -99,7 +99,7 @@ cdef class RadiusNeighbors{{name_suffix}}(BaseDistanceReducer{{name_suffix}}):
 cdef class EuclideanRadiusNeighbors{{name_suffix}}(RadiusNeighbors{{name_suffix}}):
     """EuclideanDistance-specialized {{name_suffix}}bit implementation for RadiusNeighbors{{name_suffix}}."""
     cdef:
-        GEMMTermComputer{{name_suffix}} gemm_term_computer
+        MiddleTermComputer{{name_suffix}} gemm_term_computer
```

Comment:
This eventually will be renamed.
```suggestion
        MiddleTermComputer{{name_suffix}} middle_term_computer
```

Reply:
Should we rename it and propagate the new name in this PR or later on?

Reply:
This should be done in this PR IMO.